macro/Version.py

```diff
@@ -47,11 +47,10 @@
 def execute(macro, args):
     if args:
         version = str(args)
-        if version.lower() == 'm-turtle':
-            # TODO Change when m-turtle name is decided
+        if version.lower() == 'melodic':
             return ('<span style="background-color:#FFFF00; '
                     'font-weight:bold; padding: 3px;">'
-                    'Expected in M-Turtle</span>')
+                    'Expected in Melodic Morenia</span>')
```

Comment:
The condition must be updated too. we always only used `x-turtle`. Checking for both is fine with me if you think that is necessary.